framework/include/variables/MooseVariableFV.h

```diff
@@ -608,6 +608,13 @@ class MooseVariableFV : public MooseVariableField<OutputType>
   const FieldVariablePhiValue & _phi_neighbor;
   const FieldVariablePhiGradient & _grad_phi_neighbor;
 
+  /// A member for caching retrieved Dirichlet conditions
```

Comment:
I think we should go into details here with the reason why we need this.

Reply:
Removed this cache variable in favor of non-mutable map constructed during `initialSetup`